a8c-files.php

```diff
@@ -935,7 +935,7 @@ function a8c_files_maybe_inject_image_sizes( $data, $attachment_id ) {
 
 if ( defined( 'FILES_CLIENT_SITE_ID' ) && defined( 'FILES_ACCESS_TOKEN' ) ) {
 	// Kick things off
-	add_action( 'init', 'a8c_files_init' );
+	add_action( 'muplugins_loaded', 'a8c_files_init' );
```

Comment:
One downside to waiting till `muplugins_loaded` is that any other code that runs while loading remaining `mu-plugins` and `client-mu-plugins` will not be able to use the stream wrapper. Maybe we should just call `a8c_files_init` right away instead of waiting till a hook to avoid that?

```suggestion
	a8c_files_init();
```

Reply:
Sounds like a good idea. I just tested it on my sandboxed test site and it doesn't seem to break anything